mdal/frmts/mdal_selafin.hpp

```diff
@@ -21,20 +21,50 @@ namespace MDAL
 {
   /**
    * This class is used to read the selafin file format.
-   * The file is opened with initialize() and stay opened until this object is destructed
+   * The file is opened with initialize() and stay opened until this object is destroyed
    *
    * \note SerafinStreamReader object is shared between different datasets, with the mesh and its iterators.
    *       As SerafinStreamReader is not thread safe, it has to be shared in the same thread.
    *
    * Each record of the stream has a int (4 bytes) at the beginning and the end of the record, this int is the size of the record
```

Comment:
maybe you can improve a bit docs here ? and also this one for users https://github.com/lutraconsulting/MDAL/blob/master/docs/source/drivers/selafin.rst